code/API_definitions/EdgeCloud_LcM.yaml

```diff
@@ -61,42 +61,38 @@ security:
  
 tags:
   - name: Application
-    description: Application Management
-  - name: Application Instance 
-    description: Application Instance Management
+    description: Application and Application Instance Lice Cycle Management
   - name: Edge Cloud
-    description: Edge Cloud Nodes Available
+    description: Edge Cloud Zones Available
  
 paths:
-  /appInstance:
-    post: 
-      tags:
-        - Application Instance
-      summary: Instantiation of an Application
-      description: Ask the OP to instantiate an application to one or several Edge Cloud Nodes with an Application as an input and an Application Instance as the output. 
-      operationId: appInstantiation
+  /app:
+    post:
+      tags:  
+        - Application
+      summary: Submit application metadata to the Operator Platform.                     
+      description: Contains the information about the application to be instantiated
+      operationId: submitApp
       requestBody:
-        description: Details regarding where the application should be instantiated
+        description: The Application Provider request contains mandatory criteria (e.g. required CPU, memory, storage, bandwidth) and optional parameters. 
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/AppInstantiation'
-        required: true      
+              $ref: '#/components/schemas/AppManifest'
```

Comment:
Maybe this will be confusing because an `App` schema may have more information like `appId`. But in POST /app Developers are just sending the information of their app in an `AppManifest`

Reply:
I don't see an App component definition, though. So perhaps the AppManifest component should be renamed to App? Or is it expected that there is an App component definition which includes AppManifest as part of it? It would be confusing if the /app path creates an AppManifest instead of an App, if the plan is to include an App component.

Reply:
@gainsley 
The approach for `POST /app` is that developers can send the information about their application to the Operator Platform. We used to name this information `artifact` but we change it to `AppManifest` to make it more understandable. So when developers send the application information, Operator Platform checks if all the parameters are suitable (download the image from the repository successfully), and if everything went well, OP store that information and creates `appId` that is the identifier used to retrieve the information of the application (e.g. Developer are not sure about the version, they can use appId to get the information).

Perhaps, what you are saying makes sense in terms that an `App` component needs to exist in order to retrieve all the information related to the application not only what the `Developers` initially sends to the OP but also what OP has done with the information (creating the `appId`). In that case, we can create an `App` component that contains `AppManifest` and `AppID`, that will be used in the GET /app/{appId} response.